multimodal-search-pdf/README.md

```diff
@@ -37,6 +37,18 @@ What's included in this example:
   * [Community](#community)
   * [License](#license)
     
+<details>
+ <summary>How to clone this repo❓</summary>
+ 
+ Because this repo contains many [more example projects](https://github.com/jina-ai/examples/), if you want to clone only this example, you can do this as following
+ 
+ ```bash
+ git clone --depth 1 --filter=blob:none --sparse https://github.com/jina-ai/examples
+ git sparse-checkout set multimodal-search-pdf
```

Comment:
```suggestion
cd examples
git sparse-checkout set multimodal-search-pdf
```

Reply:
I have added the `cd examples` now in the latest commit